my_enumerable.rb

```diff
@@ -0,0 +1,17 @@
+module MyEnumerable
+  def all?
+    each { |num| return false unless yield num }
+    true
+  end
+
+  def any?
+    each { |num| return true if yield num }
+    false
+  end
```

Comment:
- [ ] `Any` method returns true if the block ever returns a value other than false or nil check this and do your own test before submitting example:
`list = MyList.new(1, 2, 3, 4)`
`list.any? {|e| e == 5}`
`false`

if I do
`list.any? {|e| e == 2}`
`true`